state/state.go

```diff
@@ -27,40 +28,51 @@ func calcValidatorsKey(height int64) []byte {
 	return []byte(cmn.Fmt("validatorsKey:%v", height))
 }
 
+func calcConsensusParamsKey(height int64) []byte {
```

Comment:
TODO: actually use this to store the historical consensus params.

Alternatively, just stick the ConsensusParams directly in the header. But that's probably ridiculous.